frontend/containers/investor-form-pages/types.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  UseFormRegister,
+  RegisterOptions,
+  FieldPath,
+  FieldErrors,
+  Control,
+  UseControllerProps,
+  UseFormGetValues,
+  UseFormClearErrors,
+  UseFormSetValue,
+  UseFormResetField,
+  Path,
+  UseFormSetError,
+} from 'react-hook-form';
+
+import { Enum } from 'types/enums';
+import { InvestorForm } from 'types/investor';
+
+export type InvestorFormPagesProps = {
+  /** React Hook Form's `register` function */
+  register: UseFormRegister<InvestorForm>;
+  /** Options for React Hook Form's `register` function */
+  registerOptions?: RegisterOptions<InvestorForm, FieldPath<InvestorForm>>;
+  /** React Hook Form's state 'errors' function */
+  errors: FieldErrors<InvestorForm>;
+  /** Options for React Hook Form's `control` function */
+  control: Control<InvestorForm, FieldPath<InvestorForm>>;
+  /** Options for React Hook Form's `control` function options */
+  controlOptions?: UseControllerProps<InvestorForm, FieldPath<InvestorForm>>['rules'];
+  /** React-hook-form useForm getValues */
+  getValues?: UseFormGetValues<InvestorForm>;
+  /** React-hook-form useForm setValues */
+  setValue?: UseFormSetValue<InvestorForm>;
+  /** React-hook-form useForm setValues */
+  setError?: UseFormSetError<InvestorForm>;
+  /** React-hook-form useForm clearErrors */
+  clearErrors?: UseFormClearErrors<InvestorForm>;
+  /** React-hook-form useForm resetField */
+  resetField?: UseFormResetField<InvestorForm>;
+};
+
+export type ProjectDescriptionProps = InvestorFormPagesProps & {
+  project_development_stage: Enum[];
+  category: Enum[];
+  target_group: Enum[];
+};
+
+export type ImpactProps = Omit<InvestorFormPagesProps, 'control'> & {
+  impacts: Enum[];
+};
+
+export type FundingProps = InvestorFormPagesProps & {
+  ticket_sizes: Enum[];
+  instrument_type: Enum[];
```

Comment:
The name of these variables are quite inconsistent. Here you use the snake case but below (in `InvestmentInformationProps`), you use the camel case.